src/Shared/NativeMethodsShared.cs

```diff
@@ -137,6 +137,39 @@ private enum eDesiredAccess : int
             PROCESS_ALL_ACCESS = SYNCHRONIZE | 0xFFF
         }
 
+        internal enum LOGICAL_PROCESSOR_RELATIONSHIP
+		{
+			RelationProcessorCore,
+			RelationNumaNode,
+			RelationCache,
+			RelationProcessorPackage,
+			RelationGroup,
+			RelationAll = 0xffff
+		}
+
+        internal struct SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX
+        {
+            public LOGICAL_PROCESSOR_RELATIONSHIP Relationship;
+            public uint Size;
+            public GROUP_RELATIONSHIP Group;
+        }
+
+        internal unsafe struct GROUP_RELATIONSHIP
+        {
+            private byte MaximumGroupCount;
```

Comment:
There was a build error related to the MaximumGroupCount:
`error CS0169: The field 'NativeMethodsShared.GROUP_RELATIONSHIP.MaximumGroupCount' is never used`